docs/InstallUpdatedReactNativeCLI.md

```diff
@@ -1,9 +0,0 @@
-## Overview
```

Comment:
@madhavarshney, I tried the approach you mentioned. Works great, thanks!
I suggest not to remove this document, just leave a link to in in `CreateNewApp.md`, as to alternative way.

Reply:
I don't think it's needed because the other approach is much easier and has greater control over the version of `react-native-desktop` to install.  Would you mind elaborating on when this can be useful?  The one case I can think of is when you don't want to specify `--version`, but that's more of an edge case in my opinion.

Reply:
@madhavarshney, I think about a bright beautiful future when some people develop a lot of desktop apps on the same machine and want to have adjusted `react-native-cli` installed to type fewer characters on project initialization :)
Also, I think that if adjusted `react-native-cli` can be installed globally, we should at least mention this possibility to not forget that it exists :)

Reply:
In one sense, yes.  However, when we refactor the repository to be an extension of `react-native` rather than a fork and full implementation, the adjusted global CLI wouldn't exist, and instead `rnpm-plugin-desktop` would have to be installed in whichever repository `desktop` support needs to be added to.  For now I will leave the file as it is.

Reply:
@madhavarshney, agree, when we refactor the repository we can get rid of this part of documentation since it won't be usable at all. But for now please leave this doc and link to it, "just in case".

Reply:
Done, please take a look.